src/nfc/clf/pn532.py

```diff
@@ -404,16 +404,40 @@ def init(transport):
                 elif transport.port == "/dev/ttyS0":
                     log.debug("ttyS0 can only do 115.2 kbps")
                     change_baudrate = False  # RPi 'mini uart'
```

Comment:
I think this change_baudrate assignment is no longer needed.

Reply:
yes this is not needed